mods/blocky_spoilers/blocky_spoilers.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Reddit style spoilers",
+    "namespace": "blocky-spoilers",
+    "author": "Pamasich",
+    "version": "1.0.0",
+    "label": "Alternate spoilers",
+    "desc": "Spoilers on kbin/mbin are implemented as an element that can be collapsed and expanded. This is different from the usual approach where the spoiler text is shown in full but obscured by a black bar until hovered over or clicked.\n\nThis addon changes spoilers to work like on other platforms, which you might be more familiar with.\n\nBy default you'll be able to toggle whether a spoiler is obscured by clicking on its header (the always visible text). Optionally, hovering is also supported to temporarily show a spoiler. But you'll still have to click to make it stick.",
```

Comment:
"But you'll still have to click to make it stick." The subject of the prior sentence is "hovering", but the subject of this sentence is "the spoiler". Also, "by default" seems unnecessary here, since there is no alternate mode for the click behavior itself, and hovering doesn't itself modify the clicking.

Suggested wording based on my understanding (also rearranged some clauses):

Click a spoiler's header (the always visible text) to toggle whether the spoiler's content should be permanently visible (sticky). This behavior is off by default. Click once to turn on, then click again to turn off. Optionally, hovering is also supported to temporarily show/hide spoiler content, provided the spoiler is not currently set to sticky.

Reply:
> "But you'll still have to click to make it stick." The subject of the prior sentence is "hovering", but the subject of this sentence is "the spoiler".

I meant it more as the subject being the change to whether the spoiler is visible. But yeah, that's not clear.

> Also, "by default" seems unnecessary here, since there is no alternate mode for the click behavior itself

Yeah, agree.

> Click a spoiler's header (the always visible text) to toggle whether the spoiler's content should be permanently visible (sticky). This behavior is off by default. Click once to turn on, then click again to turn off.

I think this is a bit too ELI5. The "toggle" and "off by default" already implies that clicking once turns it on and then clicking it again turns it off again. I also don't think specifying the default state is necessary, as it's just logical in my opinion that a spoiler would be hidden by default. When I read this, it sounds to me more like the ability to click is off by default.

I'll see what I can do to improve the description. I should also mention the lock icon as an indicator of whether the spoiler is sticky or not once that's implemented, which would further reduce the need to define the default state.

Reply:
Yeah, I agree with that. I was just trying to get the salient points out and list them all one by one, but perhaps I went overboard.

I also agree that the description could greatly be streamlined when cosmetic/visual indicators are added.